### PR TITLE
Point cloud

### PR DESCRIPTION
**Pull request by clementcolin**

### CODE REVIEW

File: src/Templates/AllWidget/AllWidget.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ export class AllWidget {
       this.loadConfigFile(path).then(() => {
         // Use the stable server
         _this.addLogos();
-
+
```
Comment (EricBoix) [inline src/Templates/AllWidget/AllWidget.js:45]: Does this new line pass the eslint ?

File: src/Templates/AllWidget/AllWidget.js [hunk 2]
```diff
@@ -477,37 +477,41 @@ export class AllWidget {
...
-      materialProps: { opacity: 1, color: color },
-    });
-
-    $3DTilesManager.addEventListener(
-      TilesManager.EVENT_TILE_LOADED,
-      function (event) {
-        $3DTilesManager.setStyleToTileset('default');
-        $3DTilesManager.applyStyles();
+    
+    if(!layer['pc_size']){
```
Comment (EricBoix) [inline src/Templates/AllWidget/AllWidget.js:498]: Is it obvious to what layer['pc_size'] corresponds !?
Maybe a little comment on what the code is achieving in this block ?

File: src/Templates/AllWidget/AllWidget.js [hunk 3]
```diff
@@ -571,15 +575,14 @@ export class AllWidget {
...
       max_y
     );
-
     // Get camera placement parameters from config
     let coordinates = this.extent.center();
     if (
       this.config['camera']['position']['x'] &&
       this.config['camera']['position']['y']
-    ) {
+    ) {
```
Comment (EricBoix) [inline src/Templates/AllWidget/AllWidget.js:583]: Coding style ok ?

**Review by EricBoix — CHANGES_REQUESTED** [review]

File: src/Templates/AllWidget/AllWidget.js (hunk 1 above)
Comment (clementcolin) [reply src/Templates/AllWidget/AllWidget.js:45]: Seems to, yes

File: src/Templates/AllWidget/AllWidget.js (hunk 2 above)
Comment (clementcolin) [reply src/Templates/AllWidget/AllWidget.js:498]: Done

File: src/Templates/AllWidget/AllWidget.js (hunk 3 above)
Comment (clementcolin) [reply src/Templates/AllWidget/AllWidget.js:583]: My eslint made this coding style when formatting the file using it ... I suppose this is ok then ?

**Comment by clementcolin** [comment]
Now handling point cloud layer properly, instead of having a mystic global parameter running through UD-Viz

**Review by EricBoix — APPROVED** [review]

**Pull request merged**